### ISSUE TITLE
Import `computed.bool` from the right package

### ISSUE THREAD

**Issue by aoumiri** [adopted-ember-addons/ember-moment#340]
With the latest release of[ ember `3.27.0`](https://github.com/emberjs/ember.js/blob/master/CHANGELOG.md#v3270-May-3-2021), it's no longer possible to use `computed.bool` like that:

```js
import { computed } from '@ember/object';

// ...
  someAttribute: computed.bool('someTrackedProperty'),
// ...
```

But instead we should do:

```js
import { bool } from '@ember/object/computed';

// ...
  someAttribute: bool('someTrackedProperty'),
// ...
```

Otherwise, this is the error we get: `Uncaught TypeError: Ember.computed.bool is not a function`.

**Comment by aoumiri** [comment]
Hello @stefanpenner, any idea on how to make the pipeline pass?
It looks like the node version it's using is different from the one expected.

**Comment by schorke** [comment]
@aoumiri I don't understand why this error exists (we have the same error) while `Ember.computed.bool` is only [deprecated](https://github.com/emberjs/ember.js/blob/v3.27.0/packages/%40ember/object/index.js#L83) in Ember 3.27.0 ...

Did I miss something ?

**Comment by timmyomahony** [comment]
I'm encountering this with `ember-moment` `8.0.1` on `ember-source` `3.28.0` running the following in a template:

```
{{moment-format (now) 'YYYY'}}
```

Throws the error:

```
Uncaught TypeError: Ember.computed.bool is not a function
    at Module.callback (-base.js:10)
    at Module.exports (loader.js:106)
    at Module._reify (loader.js:143)
    at Module.reify (loader.js:130)
    at Module.exports (loader.js:104)
    at Module._reify (loader.js:143)
    at Module.reify (loader.js:130)
    at Module.exports (loader.js:104)
    at requireModule (loader.js:27)
    at Class._extractDefaultExport (index.js:462)
```

Which seems to relate to here:

https://github.com/stefanpenner/ember-moment/blob/382ddd1238c7bcab2f66225830d18aca5364c2f6/addon/helpers/-base.js#L9

More details on my env:

```
DEBUG: -------------------------------
index.js:198 DEBUG: Ember             : 3.28.0
index.js:198 DEBUG: Ember Data        : 3.27.1
index.js:198 DEBUG: jQuery            : 3.6.0
index.js:198 DEBUG: Ember Simple Auth : 3.1.0
index.js:198 DEBUG: -------------------------------
```

**Comment by timmyomahony** [comment]
Just to note, the PR from here fixes this for me:

https://github.com/stefanpenner/ember-moment/pull/351

You can get it with:

```
yarn add stefanpenner/ember-moment#351/head
```

**Comment by aoumiri** [comment]
@jasonmit @stefanpenner can anyone of you have a look at this PR? Or any maintainer 🙏

Thanks 😊

**Comment by aoumiri** [comment]
Ember bug or not, I think it's still worth fixing now if it will be deprecated. Rather than waiting for the next major. WDYT ?

**Comment by esbanarango** [comment]
https://github.com/adopted-ember-addons/ember-moment/pull/351#issuecomment-914650169

**Comment by NullVoxPopuli** [comment]
Fix released in 8.0.2
